msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/AccountState.kt

```diff
@@ -194,14 +209,16 @@ class AccountState private constructor(
     interface GetAccessTokenCallback : Callback<GetAccessTokenResult>
 
     /**
-     * Retrieves the access token for the currently signed in account from the cache.
+     * Retrieves the access token for the currently signed in account from the cache. If multiple
```

Comment:
This comment doesn't make sense, as this method doesn't accept a scopes parameter.
Also, shouldn't this method be returning a token for the default scopes? If so, that should be made clear from the comments.

@nilo-ms  can you review this based on your technical proposal please.

Reply:
I suggest adding a comment like:
`Retrieves an access token with default OIDC scopes as permission`

Reply:
Done